### ISSUE TITLE
The Qiskit Times Newsletter

### ISSUE THREAD

**Issue by HuangJunye** [qiskit-advocate/qiskit-advocate-projects#15]
# Abstract

Run a monthly / bi-monthly The Qiskit Times newsletter for Qiskit advocates.

# Description 

We have a semi-regular newsletter for advocates called [The Qiskit Times](https://qiskit.slack.com/archives/GMT338B52/p1591726883238000). It contains the latest updates of Qiskit features, events and projects that Qiskit advocates have been doing. Qiskit advocate program is all about community and open source. We want to open up the opportunity of running such newsletter to all advocates. Comics? Crosswords? All ideas are welcome! Can't wait to see what you guys can come up with. 

# Members

 - @0sophy1 
 - @githubhandle - Slack: `@slackhandle` email: `example@example.com`
 - Qiskit mentor: @BrianIngmanson 

# Deliverable

Monthly or bi-monthly email newsletter to send to mail boxes of all advocates.

# GitHub repo

**Comment by 0sophy1** [comment]
I want to join! :)

**Comment by HuangJunye** [comment]
Great, thank you Sophie! I added you as a member.

**Comment by 0sophy1** [comment]
I think people will be getting the attention of the newsletter's importance after we publish our first one. :)

How about starting our small one steps for the progress? The goal is to publish our first one at the end of October.

**Comment by starktech23** [comment]
Agreed! The next newsletter by the community team should be arriving soon, that should help with getting the attention of Advocates and hopefully have a few more members on-board. 

In the meanwhile we can start planning out resource boards, platforms for drafting etc. and aim for an end-of-October release. Having helped with two editions of the newsletter I'd be happy to contribute to the team as well!

**Comment by 0sophy1** [comment]
Received "Qiskit Newsletter" :) So, what should we start with? I think we can follow the category of previous qiskit times. How about making a shared document or making a workspace for gathering news for the category?

**Comment by HuangJunye** [comment]
@0sophy1 Let me get Brian in the loop and become a mentor for this project. He created the previous editions for The Qiskit Times. Will get back to you soon.

**Comment by BrianIngmanson** [comment]
Hey all, I'll gladly be a mentor for this project :)

**Comment by BrianIngmanson** [comment]
[The Qiskit Times - Oct 2020.docx](https://github.com/qiskit-community/qiskit-advocate-projects/files/5341079/The.Qiskit.Times.-.Oct.2020.docx)

Here's the previous source file we use to create an edition

**Comment by HuangJunye** [comment]
This is awesome @BrianIngmanson. Thanks for mentoring this project. Do you think it's a good idea to work on the project on Google Docs? I think it's probably easily to collaborate.

**Comment by 0sophy1** [comment]
> Hey all, I'll gladly be a mentor for this project :)

I think it is time to remove the "mentor needed" tag :)

**Comment by 0sophy1** [comment]
Hi, I'm sorry. I don't think I'm the right person for this project. I will unassign myself. Thank you always for your support for the Qiskitters. Please delete me from the participants.
